Python/HAT/draw.py

```
@@ -1,47 +1,134 @@
 import numpy as np
 import matplotlib.pyplot as plt
+import networkx as nx
 
+from scipy.spatial.distance import pdist
+
+import matplotlib.pyplot as plt
```

Ruff (0.8.2)

Redefinition of unused `plt` from line 2

Remove definition: `plt`

(F811)